summary.qmd

```diff
@@ -4,35 +4,39 @@ title: "What is Open Source?"
+The [Open Source Initiative](https://opensource.org/osd){target="_blank"} delineates a number of characteristics of what it means to be **open source**, including free distribution, available source code, and the ability to create derived works.
 
--   Does the cost matter?
--   Does the ability to review the code matter?
--   Does the ability to reuse the code matter?
--   Does the ability to modify the code matter?
+According to [Merriam-Webster](https://www.merriam-webster.com/dictionary/open-source){target="_blank"}, open source is “having the source code freely available for possible modification and redistribution”.
 
-## How to Contribute
+[Wikipedia](https://en.wikipedia.org/wiki/Open-source_software){target="_blank"} defines open-source software (OSS) as a computer software that is released under a license in which the copyright holder grants users the rights to use, study, change and distribute the software and its source code to anyone and for any purpose.
 
-Contribute to the discussion here in GitHub Discussions:\
-[What is open source?](https://github.com/phuse-org/OSTCDA/discussions/1){target="_blank"}
+All three of these sources refer to the ability to see the code, use the code, change the code, and share the code. And to be able to do so without any cost.
 
-## Guidance
+### How is this enabled?
 
--   Provide your thoughts and perspectives
+Fundamentally, the code is owned by someone - by some entity, whether an individual, an organization, or a group of other entities. The owner of the code have the right to disseminate the code openly and describe how it can be used by others. This is accomplished through a license agreement (discussed in another section). For any piece of **open source code**, the owner of that code has decided to release it publicly with an associated license that defines how others may use or modify the code.
 
--   Provide references to articles, webinars, presentations (citations, links)
+In a paper entitled \*[Open Source Technologies in Clinical Research](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2020/Connect/US/Virtual/PAP_TT10.pdf){target="_blank"}, as part of the 2020 PHUSE US Connect, Bruce Wienckowski et al. write that "open source software includes permission to use the source code, design documents, or content of the product." This is in contrast to proprietary software which is not typically distributed free of charge and "typically has licensed limitations on how that software can be used."
 
--   Be respectful in this community
+Data submitted for review by a health authority to approve a medicine, therapy, or vaccine into the market it governs are expected to be ***accurate***, ***reproducible***, and ***traceable*** from the point of collection to the final analytic report. The ability to demonstrate the integrity of all data transformations has been perceived to be more challenging with open source solutions (relative to proprietary or commercial solutions).
```

Comment:
"has been perceived to be more challenging with open source solutions" - citation needed. 
In contrast, it would seem *more* difficult to show accuracy, reproducibility and traceability with proprietary solutions if the license cost for those systems means that a large proportion of the population cannot perform that validation. If you have a reference to support the statement, fine, but there's surely a paradox there.